CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ t7800-difftool.sh
 t9902-completion.sh
 t9903-bash-prompt.sh
 t5310-pack-bitmaps.sh
+
+On windows when building with shared libraries
+don't forget to append their location to the PATH variable.
```

Comment:
This should be squashed into the first commit.

Reply:
okay